app.js

```diff
@@ -58,31 +60,40 @@ const displayThreeProducts = () => {
     radio2.value = product2.id;
     radio3.value = product3.id;
 
+    image1.src = product1.image;
```

Comment:
To use these you're going to need to put ids on your image tags, and you'll have to bring them into your app.js.